clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1517,11 +1517,9 @@ void handleFailure(Throwable throwable) {
     @Override
     public DescribeConfigsResult describeConfigs(Collection<ConfigResource> configResources, final DescribeConfigsOptions options) {
         final Map<ConfigResource, KafkaFutureImpl<Config>> unifiedRequestFutures = new HashMap<>();
-        final Map<ConfigResource, KafkaFutureImpl<Config>> brokerFutures = new HashMap<>(configResources.size());
-
         // The BROKER resources which we want to describe.  We must make a separate DescribeConfigs
```

Comment:
I guess we should update or remove this comment?

Reply:
@hachikuji thanks for the review. I believe that comment about ``unifiedRequestResources`` is still valid, see:

```java
        // The non-BROKER resources which we want to describe.  These resources can be described by a
        // single, unified DescribeConfigs request.
        final Collection<Resource> unifiedRequestResources = new ArrayList<>(configResources.size());

        for (ConfigResource resource : configResources) {
            if (resource.type() == ConfigResource.Type.BROKER && !resource.isDefault()) {
                brokerFutures.put(resource, new KafkaFutureImpl<Config>());
            } else {
                unifiedRequestFutures.put(resource, new KafkaFutureImpl<Config>());
                unifiedRequestResources.add(configResourceToResource(resource));
            }
        }
```